testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/miniapptabs/fragments/MiniAppDisplayFragment.kt

```diff
@@ -315,6 +320,24 @@ class MiniAppDisplayFragment : BaseFragment(), PreloadMiniAppWindow.PreloadMiniA
                     AppPermission.getDeviceRequestCode(miniAppPermissionType)
                 )
             }
+
+            override fun sendJsonToHostApp(
+                jsonStr: String,
+                onSuccess: (String) -> Unit,
```

Comment:
(optional) you also can use `onSuccess` to send a confirmation message to MiniApp. `onSuccess` will display an alert dialog like this with invoking `onSuccess(jsonStr + "has been found from MiniApp.")`

![Screen Shot 2022-12-02 at 12 17 51](https://user-images.githubusercontent.com/66930373/205207550-a93b8a15-aa69-4f95-b8a7-73ec183f98e5.png)